{"name": "Test login with incorrect login credentials with parameters -- @1.1 Credentials", "status": "broken", "statusDetails": {"message": "AttributeError: 'Context' object has no attribute 'driver'\n", "trace": "  File \"/home/oleg/PycharmProjects/bdd/.venv/lib/python3.10/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/oleg/PycharmProjects/bdd/.venv/lib/python3.10/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/login_steps_with_parameters.py\", line 13, in step_impl\n    SeleniumHelper(context.driver).open_page(LOGIN_URL)\n  File \"/home/oleg/PycharmProjects/bdd/.venv/lib/python3.10/site-packages/behave/runner.py\", line 321, in __getattr__\n    raise AttributeError(msg)\n"}, "steps": [{"name": "When user input wrong \"zapel\" and password \"zapel\"", "status": "broken", "statusDetails": {"message": "AttributeError: 'Context' object has no attribute 'driver'\n", "trace": "  File \"/home/oleg/PycharmProjects/bdd/.venv/lib/python3.10/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/oleg/PycharmProjects/bdd/.venv/lib/python3.10/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/login_steps_with_parameters.py\", line 13, in step_impl\n    SeleniumHelper(context.driver).open_page(LOGIN_URL)\n  File \"/home/oleg/PycharmProjects/bdd/.venv/lib/python3.10/site-packages/behave/runner.py\", line 321, in __getattr__\n    raise AttributeError(msg)\n"}, "attachments": [{"name": ".text", "source": "1f0eb8db-d2f3-48b6-b935-efd3b2590015-attachment.txt", "type": "text/plain"}], "start": 1727621663082, "stop": 1727621663083}, {"name": "Then error message will come with parameters", "status": "skipped", "start": 1727621663083, "stop": 1727621663083}], "parameters": [{"name": "login_id", "value": "zapel"}, {"name": "password", "value": "zapel"}], "start": 1727621663081, "stop": 1727621663083, "uuid": "3c4c268a-eb13-4463-b868-56bc146f067e", "historyId": "957e6aa493a3bfb3ee142d79f26263a2", "fullName": "Facebook login functionality with parameters: Test login with incorrect login credentials with parameters", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Facebook login functionality with parameters"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}